cirq/ops/linear_combinations_test.py

```diff
@@ -172,5 +181,383 @@ def test_in_place_manipulations(gates):
+         [0, 0, 0, 1.1, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0.1, -1j, 0, 0],
+         [0, 0, 0, 0, 1j, 0.1, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0.1, -1j],
+         [0, 0, 0, 0, 0, 0, 1j, 0.1],
+     ])),
+    ({
+        cirq.I(q2): 0.1,
+        cirq.ControlledGate(cirq.Y).on(q0, q1): 1
+    },
+     np.array([
+         [1.1, 0, 0, 0, 0, 0, 0, 0],
+         [0, 1.1, 0, 0, 0, 0, 0, 0],
+         [0, 0, 1.1, 0, 0, 0, 0, 0],
+         [0, 0, 0, 1.1, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0.1, 0, -1j, 0],
+         [0, 0, 0, 0, 0, 0.1, 0, -1j],
+         [0, 0, 0, 0, 1j, 0, 0.1, 0],
+         [0, 0, 0, 0, 0, 1j, 0, 0.1],
+     ])),
+    ({
+        cirq.I(q0): 0.1,
+        cirq.FREDKIN(q1, q2, q3): 1
+    },
+     np.array([
+         [1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0.1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 1, 0.1, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 1.1, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 1.1, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 1.1, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1.1, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1.1, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1.1, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0.1, 1, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0.1, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1.1],
+     ])),
+    ({
+        cirq.I(q1): 0.1,
+        cirq.FREDKIN(q0, q2, q3): 1
+    },
+     np.array([
+         [1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 1.1, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 1.1, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0.1, 1, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0.1, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1.1, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1.1, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0.1, 1, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0.1, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1.1],
+     ])),
+    ({
+        cirq.I(q2): 0.1,
+        cirq.FREDKIN(q0, q1, q3): 1
+    },
+     np.array([
+         [1.1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
```

Comment:
It's quite hard to see the non-zero elements in these grids...

Maybe something crazy like `o = 0` and using `o`? Or `_`? Not necessary.

Reply:
Done.

Used `_` and I think the result is really good. Perhaps we should use it when we hardcode large sparse matrices in tests more generally.